lib/postcode_loader.rb

```diff
@@ -58,13 +58,16 @@ def validate_csv_headers!
   def defer_local_authority_integrity_checks_until_commit!
     postcode_local_authority_foreign_key = "fk_rails_7ab3384eab"
     office_local_authority_foreign_key = "fk_rails_5a2ab5b59d"
+    served_areas_local_authority_foreign_key = "fk_rails_90a03e3955"
```

Comment:
Is it ok to reference foreign keys by their generated name? I guess they will always be the same because they are in the migration scripts?

Reply:
I think so, I've not figured out a better way to do it!